CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.1.1 (Unreleased)
+
+Bugfixes and corrections.
```

Comment:
Open an issue for this fix and give issue  number here

Reply:
Issue is opened already and attached to the PR also. Do you mean to mention it in ChangeLog ..?

Reply:
mentioned issue number in CHANGELOG file